### PR TITLE
Add flag to contact and room payload.

### PR DESCRIPTION
**Pull request by su-chang**
Related issue: https://github.com/wechaty/wechaty-puppet/issues/90

### CODE REVIEW

File: src/schemas/contact.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export interface ContactPayload {
   city?      : string,
   friend?    : boolean,
   province?  : string,
+  isDelete?  : boolean,
```
Comment (windmemory) [inline src/schemas/contact.ts]: better to name this as `isDeleted`

File: src/schemas/room.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ export interface RoomPayload {
 
   topic        : string,
   avatar?      : string,
+  isDelete?    : boolean,
```
Comment (windmemory) [inline src/schemas/room.ts]: ditto

**Comment by su-chang** [comment]
@huan ping

**Comment by su-chang** [comment]
No need to add `isDeleted` attribute for `roomPayload`.

Detail info: #90

**Pull request closed without merging**